subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -198,102 +198,151 @@ Let's say you wanted to declare a dependency on the link:https://tomcat.apache.o
 
-Other examples of dependency exclusions can be found in the reference for the api:org.gradle.api.artifacts.ModuleDependency[] or api:org.gradle.api.artifacts.dsl.DependencyHandler[] classes.
+++++
+<sample id="force-per-configuration" dir="userguide/dependencies/forcingDependencyVersionPerConfiguration" title="Enforcing a dependency version on the configuration-level">
+    <sourcefile file="build.gradle" snippet="force-per-configuration"/>
+</sample>
+++++
 
-[[sec:optional_attributes]]
-==== Optional attributes
+[[sub:disabling_resolution_transitive_dependencies]]
+==== Disabling resolution of transitive dependencies
 
-All attributes for a dependency are optional, except the name. Which attributes are required for actually finding dependencies in the repository will depend on the repository type. See <<sec:declaring_repositories>>. For example, if you work with Maven repositories, you need to define the group, name and version. If you work with filesystem repositories you might only need the name or the name and the version.
+By default Gradle resolves all transitive dependencies specified by the dependency metadata. Sometimes this behavior may not be desirable e.g. if the metadata is incorrect or defines a large graph of transitive dependencies. You can tell Gradle to disable transitive dependency management for a dependency by setting api:org.gradle.api.artifacts.ModuleDependency#setTransitive(boolean)[] to `true`. As a result only the main artifact will be resolved for the declared dependency.
 
 ++++
-<sample id="dependenciesWithEmptyAttributes" dir="userguide/artifacts/externalDependencies" title="Optional attributes of dependencies">
-                <sourcefile file="build.gradle" snippet="dependencies-with-empty-attributes"/>
-            </sample>
+<sample id="disabling-transitive-dependency-resolution" dir="userguide/dependencies/disablingTransitiveDependencyResolution" title="Disabling transitive dependency resolution for a declared dependency">
+    <sourcefile file="build.gradle" snippet="transitive-per-dependency"/>
+</sample>
 ++++
 
-You can also assign collections or arrays of dependency notations to a configuration:
+[NOTE]
+====
+Disabling transitive dependency resolution will likely require you to declare the necessary runtime dependencies in your build script which otherwise would have been resolved automatically. Not doing so might lead to runtime classpath issues.
+====
+
+A project can decide to do away without transitive dependency resolution completely. You either don't want to rely on the metadata published to the consumed repositories or you want to gain full control over the dependencies in your graph. Transitive dependency resolution can be disabled on a configuration-level for any configuration returning `true` for api:org.gradle.api.artifacts.Configuration#isCanBeResolved()[]. For more information, see api:org.gradle.api.artifacts.Configuration#setTransitive(boolean)[].
```

Comment:
"do away without" is a bit off to my American mind. "Disable" is [Strunk and White](https://en.wikipedia.org/wiki/The_Elements_of_Style) IMO

Reply:
"returning `true` for ..." probably can be shortened to "resolvable" once we clarify what that is above.

Reply:
> "do away without" is a bit off to my American mind. "Disable" is Strunk and White IMO

done